recipes/sqlathanor/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "sqlathanor" %}
+{% set version = "0.7.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sqlathanor-{{ version }}.tar.gz
+  sha256: 301bd78ef7e56c02ed9f4a5795d9cc89d3c3a18c646b7c0dc42bd38d78264a11
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+  noarch: python
+
+requirements:
+  host:
+    - python >=3.7,<4
+    - pip
+  run:
```

Comment:
Looking at the upstream `setup.py`, for python `>=3.7`, it looks like the version requirements for some of the packages are incorrect:

```yaml
- sqlalchemy >=0.10
- pyyaml >=3.13
```